subiquity/models/identity.py

```diff
@@ -13,10 +13,10 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import logging
-
 import attr
+import logging
```

Comment:
so the import pattern I use in subiquity (which I don't think I've written down anywhere, but iirc I got this habit from launchpad) is:

```
<stdlib modules, in alphabetical order>
<blank line>
<third party modules, in alphabetical order>
<blank line>
<other packages from the same project, in alphabetical order>
<blank line>
<imports from the same package>
```

'attr' is a third party module, so that's why it was where it was.